Doc/library/multiprocessing.rst

```diff
@@ -1837,8 +1837,8 @@ Running the following commands creates a server for a single shared queue which
 remote clients can access::
 
    >>> from multiprocessing.managers import BaseManager
-   >>> import queue
-   >>> queue = queue.Queue()
+   >>> from multiprocessing import Queue
```

Comment:
I don't think `multiprocessing.Queue` is needed here. We don't communicate between different processes in the same machine so using `queue.Queue` is OK.

Reply:
thank you for the comment, what should I do now? Should I make the change and submit a new pull request?

Reply:
You don't need to submit a new pull request. You can make the requested changes and push to your existing branch. In your case, that would be the `master` branch.

You can follow the guide at https://devguide.python.org/pullrequest/#step-by-step-guide for your future pull requests.